test/python/circuit/test_number_of_qubits.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test registerless QuantumCircuit and Gates on wires"""
+
+
+from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
+from qiskit.test import QiskitTestCase
+
+
+class TestNumberOfQubits(QiskitTestCase):
+    """Test number of qubits in Quantum Circuit"""
+    def qubitless_circuit(self):

```

Comment:
Can you update these tests to start with the prefix `test_`? Also I think it would be good to have a test with multiple quantum registers in the same circuit.